sql/core/src/test/resources/sql-tests/results/group-by.sql.out

```diff
@@ -1152,16 +1152,15 @@ SELECT mode(col, 'true') FROM VALUES (-10), (0), (10) AS tab(col)
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.catalyst.ExtendedAnalysisException
+org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "DATATYPE_MISMATCH.UNEXPECTED_INPUT_TYPE",
-  "sqlState" : "42K09",
+  "errorClass" : "WRONG_NUM_ARGS.WITHOUT_SUGGESTION",
```

Comment:
Actually, the previous error messages were more clear in these 3 cases.

Reply:
Yes. If we need keep compatibility with origin `deterministicExpr`, I will restore them.